espressif/vscode-esp-idf-extension
# add esp insights framework support

## Description

Add ESP-Insights as supported framework to install and set the `idf.espInsightsPath`. It will use the setting as  `INSIGHTS_PATH` as env variable for the extension tasks.

## Type of change

Please delete options that are not relevant.
- New feature (non-breaking change which adds functionality)

## Steps to test this pull request

Provide a list of steps to test changes in this PR and required output

1. Click on "ESP-IDF: Install ESP-Insights"
2. Execute action. The repository will be cloned and the `idf.espInsightsPath `settings will be updated.
3. Click on "ESP-IDF: Show Examples" and choose ESP-Insights. Pick an example
4. Try to build the example after configuring this example as required. Some changes to idf_component.yml might be required to remove idf_component manager behavior.

- Expected behaviour:

- Expected output:

**Test Configuration**:
* ESP-IDF Version: 4.4
* OS (Windows,Linux and macOS): macOS

## Checklist
